certstrap/config-production.yaml

```diff
@@ -0,0 +1,14 @@
+---
+src-repo: https://github.com/square/certstrap
+src-branch: master
+src-private-key: ~
```

Comment:
Is a private key necessary here since this is a public repo and it's already using the https git url?